src/unix.rs

```diff
@@ -116,6 +116,10 @@ impl Client {
     }
 
     /// `--jobserver-auth=R,W`
+    ///
+    /// If either or both of these file descriptors are negative,
+    /// it means the jobserver is disabled for this process
+    /// (See ["GNU male manual: POSIX Jobserver Interaction"](https://www.gnu.org/software/make/manual/make.html#POSIX-Jobserver)).
```

Comment:
Same typo as in the other PR :)

Also, perhaps this renders nicely:

```suggestion
    /// (See ["GNU `make` manual: POSIX Jobserver Interaction"](https://www.gnu.org/software/make/manual/make.html#POSIX-Jobserver)).
```

Reply:
Must be copied from there 🤣